src/components/FirefoxReleaseVersionMarkers.svelte

```diff
@@ -1,35 +1,27 @@
 <script>
-import { getContext } from 'svelte';
-import Marker from 'udgl/data-graphics/guides/Marker.svelte';
-import { store } from '../state/store';
-import { firefoxVersionMarkers } from '../state/product-versions';
+  import { getContext } from 'svelte';
+  import { Marker } from '@graph-paper/guides';
+  import { store } from '../state/store';
+  import { firefoxVersionMarkers } from '../state/product-versions';
 
+  export let labels = true;
 
-export let labels = true;
-
-const xScale = getContext('xScale');
-
-let markers = [];
-firefoxVersionMarkers.subscribe((m) => { markers = m; });
+  const xScale = getContext('xScale');
 
+  let markers = [];
+  firefoxVersionMarkers.subscribe((m) => {
+    markers = m;
+  });
 </script>
 
-
-<g class='firefox-release-version-markers'>
-  {#if markers && markers.length}
-    {#each markers
-      .filter((d) => d.date !== undefined && d.date >= $xScale.domain()[0] && d.date <= $xScale.domain()[1]) as {label, date}, i (date)}
-      <Marker location={date}>
-        {#if labels}
-          {#if $store.productDimensions.channel === 'nightly'}
-            {label + 2}
-          {:else if $store.productDimensions.channel === 'beta'}
-            {label + 1}`
-          {:else}
-            {label}
-          {/if}
-        {/if}
-      </Marker>
-    {/each}
-  {/if}
+<g class="firefox-release-version-markers">
+  {#if markers && markers.length} {#each markers.filter((d) => d.date !==
```

Comment:
```
  18:71  error  '!==' should be placed at the beginning of the line  operator-linebreak
```

Reply:
I'll have to double-check but I think most of these issues you've commented on are a byproduct of Prettier's auto-formatting

Reply:
yep, just double-checked. My editor formatted it this way. Almost all of these errors commented on seem to be the editor fighting with eslint. Wondering what that's about.

Reply:
Something must be wrong. prettier-plugin-svelte formats this file very differently for me:

```svelte
<g class="firefox-release-version-markers">
  {#if markers && markers.length}
    {#each markers.filter((d) => d.date !== undefined && d.date >= $xScale.domain()[0] && d.date <= $xScale.domain()[1]) as { label, date }, i (date)}
      <Marker location={date}>
```

Something else might be formatting the file. If it's not that, perhaps there's an old version of prettier-plugin-svelte installed globally (try `npm ls -g prettier-plugin-svelte`) or the vscode plugin is using an old version of prettier-plugin-svelte.

Reply:
We can prioritize #480 before or after we merge this PR so that we don't run into these annoyances in the future. Either way, I'd check the configuration for the future. Something seems off.

Reply:
We can prioritize #480 either before or after we merge this PR so that formatting becomes more seamless and invisible in the future. If we'd rather merge this PR first, that's totally fine. Just ignore the ESLint problems I've commented about here. Running ESLint after formatting the files with the latest prettier-plugin-svelte actually brings the problem count down from 45 to 19 automatically.

Either way, I'd check the editor's configuration for the future. Something seems off.

Reply:
Thanks for taking a look. I'll see if something is off w/ my plugins. I agree that getting this PR sorted out is probably a higher priority.